src/main/java/appeng/capabilities/Capabilities.java

```diff
@@ -0,0 +1,48 @@
+package appeng.capabilities;
+
+
+import net.minecraft.nbt.NBTBase;
+import net.minecraft.util.EnumFacing;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.CapabilityInject;
+import net.minecraftforge.common.capabilities.CapabilityManager;
+
+import appeng.api.storage.IStorageMonitorableAccessor;
+
+
+/**
+ * Utility class that holds various capabilities, both by AE2 and other Mods.
+ */
+public class Capabilities
+{
+
+	@CapabilityInject( IStorageMonitorableAccessor.class )
+	public static Capability<IStorageMonitorableAccessor> STORAGE_MONITORABLE_ACCESSOR;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "STORAGE_MONITORABLE_ACCESSOR" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3008)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this "public static STORAGE_MONITORABLE_ACCESSOR" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1444)